docs/user-guide/deployments/deploy-on-kubernetes/greptimedb-operator-management.md

```diff
@@ -0,0 +1,218 @@
+* greptimedb-operator
+
+The greptimedb-operator is starting, use `kubectl get deployments greptimedb-operator -n greptimedb-admin` to check its status.
+```
+</details>
+
+If you want to upgrade to a specific version, you can use the following command:
+
+```bash
+helm --namespace greptimedb-admin upgrade greptimedb-operator greptime/greptimedb-operator --version <version>
+```
+
+After the upgrade is complete, you can use the following command to verify the installation:
+
+```bash
+helm list -n greptimedb-admin
+```
+
+<details>
+<summary>Expected Output</summary>
+```bash
+NAME               	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART                    	APP VERSION
+greptimedb-operator	default  	2       	2024-10-28 19:30:52.62097 +0800 CST 	deployed	greptimedb-operator-0.2.9	0.1.3-alpha.1
+```
+</details>
+
+### CRDs
+
+There are two kind of CRD that are installed with the GreptimeDB Operator: `GreptimeDBCluster` and `GreptimeDBStandalone`.
+
+You can use the following command to verify the installation:
+
+```bash
+kubectl get crd | grep greptime
+```
+
+<details>
+  <summary>Expected Output</summary>
+```bash
+greptimedbclusters.greptime.io      2024-10-28T08:46:27Z
+greptimedbstandalones.greptime.io   2024-10-28T08:46:27Z
+```
+</details>
+
+By default, the GreptimeDB Operator chart will manage the installation and upgrade of the CRDs and the users don't need to manage them manually.
+
+### Configuration
+
+The GreptimeDB Operator chart provides a set of configuration options that allow you to customize the installation, you can refer to the [GreptimeDB Operator Helm Chart](https://github.com/GreptimeTeam/helm-charts/blob/main/charts/greptimedb-operator/README.md) for more details.
+
+You can create a `values.yaml` to configure the GreptimeDB Operator chart, for example:
+
+```yaml
+image:
+  # -- The image registry
+  registry: docker.io
+  # -- The image repository
+  repository: greptime/greptimedb-operator
+  # -- The image pull policy for the controller
+  imagePullPolicy: IfNotPresent
+  # -- The image tag
+  tag: latest
+  # -- The image pull secrets
+  pullSecrets: []
+
+replicas: 2
+
+resources:
+  limits:
+    cpu: 200m
+    memory: 256Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+```
+
+You can use the following command to install the GreptimeDB Operator with the custom configuration:
+
+```bash
+helm --namespace greptimedb-admin install greptimedb-operator greptime/greptimedb-operator -f values.yaml
+```
+
+If you want to upgrade the GreptimeDB Operator with the custom configuration, you can use the following command:
+
+```bash
+helm --namespace greptimedb-admin upgrade greptimedb-operator greptime/greptimedb-operator -f values.yaml
+```
+
+:::tip
+You can use one command to install or upgrade the GreptimeDB Operator with the custom configuration:
+
+```bash
+helm --namespace greptimedb-admin upgrade --install \
+  greptimedb-operator greptime/greptimedb-operator -f values.yaml
+```
+:::
```

Comment:
please do not use `tip`, this is a feature that should be introduced in the main text.

Reply:
It's not a feature. It's like some kind of `helm` tip.